src/containers/community/CommunityPosts.js

```diff
@@ -68,25 +83,64 @@ export class CommunityPosts extends Component {
     let { location: { query }, dispatch, community, params: { id } } = this.props
     const { currentUser } = this.context
 
+    let module
+
+    if (groupUser(currentUser.id, 'In-Feed Engagement Modules') === 0) {
+      const showPopularSkills = community.memberCount >= MIN_MEMBERS_FOR_SKILLS_MODULE
```

Comment:
this can be a top-level function that just takes community and moduleChoice and returns module